App/Graphql/query/getAllNewsByUserId.graphql

```diff
@@ -1,33 +1,30 @@
-query getNewsByUserID($user_id: ID!){
-  getNewsByUserID(user_id: $user_id) {
+query getConferenceByIDQuery($id: ID!) {
```

Comment:
There is something wrong with this file, the file name is `getAllNewsByUserId.graphql` but the query is `getConferenceByIDQuery`